lib/node_modules/@stdlib/stats/base/dcumax/docs/repl.txt

```diff
@@ -3,8 +3,8 @@
     Computes the cumulative maximum of double-precision floating-point strided
     array elements.
 
-    The `N` and `stride` parameters determine which elements in `x` and `y` are
-    accessed at runtime.
+    The `N` and `stride` parameters determine which elements in the strided
```

Comment:
```suggestion
    The `N` and stride parameters determine which elements in the strided
```

"arrays" can likely now fit on the previous line. Meaning, you'll likely need to rewrap the text.